mandatory/2-logic-error.js

```diff
@@ -1,16 +1,16 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+  return word.trim();
 }
 
 function getStringLength(word) {
-  return "word".length();
+ return word.length;
 }
```

Comment:
That is correct, you cannot surround a variable with quotation mark, since JavaScript will treat it as a String.